backend/Application/appsettings.Development.json

```diff
@@ -1,7 +1,9 @@
 {
   "PostgresDatabase": {
     "ConnectionString" : "Host=localhost;Port=5432;Database=ccscan;User ID=postgres;Password=password;Include Error Detail=true;",
-    "ConnectionStringNodeCache" : "Host=localhost;Port=5432;Database=ccscan_node_cache;User ID=postgres;Password=password;Include Error Detail=true;"
+    "ConnectionStringNodeCache" : "Host=localhost;Port=5432;Database=ccscan_node_cache;User ID=postgres;Password=password;Include Error Detail=true;",
+    "QueryTimeout": "00:00:30",
+    "MigrationTimeout": "00:02:00"
```

Comment:
Why is this a reasonable number? Why do we want to limit migration time like this?

Is this migration time relevant only when we upgrade the service? In that case, won't it just fail?